src/effects/builtin/reverbeffect.cpp

```diff
@@ -73,14 +76,31 @@ EffectManifestPointer ReverbEffect::getManifest() {
     send->setMinimum(0);
     send->setDefault(0);
     send->setMaximum(1);
+
+    EffectManifestParameterPointer dryWet = pManifest->addParameter();
+    dryWet->setId("dry_wet");
+    dryWet->setName(QObject::tr("Dry/Wet"));
+    dryWet->setShortName(QObject::tr("Dry/Wet"));
+    dryWet->setDescription(QObject::tr(
+        "Mix between the input (dry) and output (wet) of the effect\n"
+        "Lowering this fades out the effect abruptly\n"
+        "Use this to adjust the amount of the effect when the effect unit is in D+W mode"));
```

Comment:
Why is the Mix knob not sufficient for this?

Reply:
The effect unit mix knob affects the whole chain.